content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
+
+To check your marker matches one of these types, you can use the `in` operator:
+
+```python
+marker, * = r.camera.see()  # Get only the closest marker 
```

Comment:
Please can we drop the "closest marker" bit here? While I agree that it's correct, it's irrelevant.

Reply:
I think explaining what the syntax is producing helps explain what's going on. I could live with changing 'closest' to 'first' perhaps?